pseudo/about.html

```diff
@@ -11,7 +11,7 @@
           <!--Add an about section-->

```

Comment:
Nice attempt @chiieche 
At the top of your lines of code, you can open your comment `<!--` then close the comment `-->` at the bottom. This will save you the pain of commenting every single line.